uktrade/directory-cms
# Kls 804 impliment openapi spec for directory cms

This ticket adds OpenAPI (Swagger) documentation to the Directory CMS repo. 

TESTING: 
- Verify /openapi/ endpoint works
- Verify /openapi/ui/ endpoint works for admin only

### Workflow

- [ X] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [ X] Jira ticket has the correct status.
- [ X] A clear/description pull request messaged added.

### Reviewing help

- [ X] Explains how to test locally, including how to set up appropriate data

### Housekeeping

- [ X] Added all new environment variables to Vault.
- [ X] Python requirements have been re-compiled.


### Merging
